evoke/src/main.cpp

```diff
@@ -134,7 +134,12 @@ int main(int argc, const char **argv)
     }
     for(auto &u : op.unknownHeaders)
     {
-        std::cerr << "Unknown header: " << u << "\e[K\n";
+        std::cerr << "Unknown header: " << u 
+#if defined(_MSC_VER)
+                  << "\n";
```

Comment:
This should actually be  on all. The "unknown headers" should be sent out by the reporter, and the Windows reporter should not be using ANSI escape codes. Not sure how this ended up here.